### PR TITLE
[Pibbble][106751296] Fix/Profile Update Fields

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: database/migrations/2015_11_23_092743_add_columns_to_users_table.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddColumnsToUsersTable extends Migration
```
Comment (ghost) [inline database/migrations/2015_11_23_092743_add_columns_to_users_table.php:6]: Nitpick expects a namespace for tests? @andela-potemuyiwa

File: database/migrations/2015_11_23_092743_add_columns_to_users_table.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function(Blueprint $table){
```
Comment (ghost) [inline database/migrations/2015_11_23_092743_add_columns_to_users_table.php]: Also space after function keyword in anonymous functions? @andela-potemuyiwa

File: database/migrations/2015_11_23_092743_add_columns_to_users_table.php (hunk 1 above)
Comment (unicodeveloper) [inline database/migrations/2015_11_23_092743_add_columns_to_users_table.php:6]: But this is a migration file not tests. Ignore that

Comment (ghost) [inline database/migrations/2015_11_23_092743_add_columns_to_users_table.php:6]: It expected space after function keyword(anonymous function) in this file.

_Oladipupo Isola | Software Developer, Andela_
_See what TechCrunch
http://techcrunch.com/2015/06/25/backed-by-spark-capital-andela-will-develop-a-continent-of-tech-talent/
and StackOverflow
http://blog.stackoverflow.com/2015/05/stack-overflow-and-andela-partner-to-provide-education-beyond-borders/
say
about us!_
(c) +234-703-302-0216 | Andela.com http://andela.com/

On Tue, Nov 24, 2015 at 6:56 AM, Prosper Otemuyiwa <notifications@github.com

Comment (unicodeveloper) [inline database/migrations/2015_11_23_092743_add_columns_to_users_table.php:6]: Ignore

File: database/migrations/2015_11_23_092743_add_columns_to_users_table.php (hunk 2 above)
Comment (unicodeveloper) [inline database/migrations/2015_11_23_092743_add_columns_to_users_table.php]: yeah..put that

File: database/migrations/2015_11_23_092743_add_columns_to_users_table.php [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('users', function(Blueprint $table){
```
Comment (unicodeveloper) [inline database/migrations/2015_11_23_092743_add_columns_to_users_table.php]: Add that

File: database/migrations/2015_11_23_092743_add_columns_to_users_table.php (hunk 2 above)
Comment (ghost) [inline database/migrations/2015_11_23_092743_add_columns_to_users_table.php]: Add space after the function keyword?

Comment (unicodeveloper) [inline database/migrations/2015_11_23_092743_add_columns_to_users_table.php]: yeah

like this:  `function (Blueprint $table)`

Comment (ghost) [inline database/migrations/2015_11_23_092743_add_columns_to_users_table.php]: I see it is in proper shape. Does Nitpick do fixing?

Comment (unicodeveloper) [inline database/migrations/2015_11_23_092743_add_columns_to_users_table.php]: Nah..and i can see that is not in proper shape. There is not space after the function keyword, just befor the opening parenthesis

Comment (ghost) [inline database/migrations/2015_11_23_092743_add_columns_to_users_table.php]: Just checked the branch I just pushed, in 2015_11_23_092743_add_columns_to_users_table.php. There is space before the open parenthesis for the two cases.

**Pull request merged**